extension/encoding/awslogsencodingextension/internal/unmarshaler/cloudtraillog/unmarshaler.go

```diff
@@ -141,56 +143,92 @@ func NewCloudTrailLogUnmarshaler(buildInfo component.BuildInfo) *CloudTrailLogUn
 }
 
 func (u *CloudTrailLogUnmarshaler) UnmarshalAWSLogs(reader io.Reader) (plog.Logs, error) {
-	decompressedBuf, err := io.ReadAll(reader)
+	bufferedReader := bufio.NewReaderSize(reader, 64*1024)
```

Comment:
How do you deduct this size? I am always a bit lost on this, I feel I should learn it as well